app/assets/javascripts/editor.js

```diff
@@ -147,3 +148,19 @@ $E = {
     else preview_btn.button('reset');
   }
 }
+
+
+
+function urlTransformer(url) {
```

Comment:
How's this supposed to work? What are we doing with the `url` argument? :)... Can you please fix this? I think you could choose a better name because I'm not sure how you're "transforming" any URL in this.

Reply:
So the caja  html_sanitation not only sanitizes for html. It also sanitizes the input to avoid javascript injection. The urlTransformer is used when a malicious attacker tries to import javascript. (e.g. <script src="badharmfulcode.com/bad.js></script>". The urlTransformer would substitute that link to "https://publiclab.org/dashboard" in this case so that the injection fails.